### PR TITLE
feat: add lnurlauth from lnbits

### PR DESCRIPTION
**Pull request by dni**
- add lnurlauth possibility
- adds, execute function to core and cli, currently only supports tags `payRequest` and `login`

### CODE REVIEW

File: lnurl/core.py [hunk 1]
```diff
@@ -52,3 +54,53 @@ def handle(
...
+        res = handle(bech32_or_address)
+    except Exception as exc:
+        raise LnurlResponseException(str(exc))
+
+    if isinstance(res, LnurlPayResponse) and res.tag == "payRequest":
+        return execute_pay_request(res, value)
+    elif isinstance(res, LnurlAuthResponse) and res.tag == "login":
+        return execute_login(res, value)
+
+    raise Exception(f"{res.tag} not implemented")  # type: ignore
```
Comment (motorina0) [inline lnurl/core.py:70]: why not:

```suggestion
    raise LnurlResponseException(f"{res.tag} not implemented")
```

File: lnurl/core.py [hunk 2]
```diff
@@ -52,3 +54,53 @@ def handle(
...
     return get(lnurl.url, response_class=response_class, verify=verify)
+
+
+def execute(bech32_or_address: str, value: str) -> LnurlResponseModel:
+    try:
+        res = handle(bech32_or_address)
+    except Exception as exc:
+        raise LnurlResponseException(str(exc))
+
+    if isinstance(res, LnurlPayResponse) and res.tag == "payRequest":
```
Comment (motorina0) [inline lnurl/core.py:65]: nitpick: no the '==' comparison first:

```suggestion
    if res.tag == "payRequest" and isinstance(res, LnurlPayResponse):
```

same below

**Review by motorina0 — APPROVED** [review]
LGTM, see comments

**Review by talvasconcelos — APPROVED** [review]

File: lnurl/core.py (hunk 2 above)
Comment (dni) [reply lnurl/core.py:65]: this throw and linting issue because it cannot find the tag in the the repsonse

**Pull request merged**